app/views/editor/ai-scenario/AIScenarioEditView.js

```diff
@@ -59,10 +59,15 @@ module.exports = (AIScenarioEditView = (function () {
     buildTreema () {
       if ((this.treema != null) || (!this.scenario.loaded)) { return }
       const data = $.extend(true, {}, this.scenario.attributes)
+      const copySchema = { ...AIScenario.schema }
+      if (this.scenario.get('mode') === 'use') {
```

Comment:
Probably a good idea to add validation in the backend itself in pre save or something